.claude/commands/issue.md

```
@@ -0,0 +1,251 @@
+
+### 3단계: 이슈 출력
+
+판단된 타입에 맞는 템플릿을 사용하여 **제목 + 본문**을 출력합니다.
+
+---
+
+## 제목 형식
+
+기존 이슈 패턴을 준수합니다:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)